formbricks/formbricks
# fix: docker env parsing after nextjs 13.4.19 upgrade

## What does this PR do?
This PR took way longer than I thought but we finally managed to get a closure on this! (hopefully)
The issues are:
- If we have a `.env`, docker compose reads it for its own (docker) configurations and the `/*` is not understood by it
- Next, the `.env.docker` was redundant hence has been removed for a cleaner and one-place-for-all `docker-compose` itself!
- The environment variables can be changed without rebuilding the image else the other env vars are passed as arguments at buildtime hence would need a rebuild.

One point to note is, if a user wants to run the docker method, they have to get rid (rename/backup) of their `.env` file.

This might also need a new release of the docker image on dockerhub with the build time values to be passed as args since we have changed the dockerfile.

## Type of change

## How should this be tested?
1. Edit the variables in the docker-compose as you like
2. `docker compose up`
3. Verify the changes

## Checklist
